wandb/sdk/launch/launch.py

```diff
@@ -26,7 +26,7 @@ def resolve_agent_config(
     api: Api,
     entity: Optional[str],
     project: Optional[str],
-    max_jobs: Optional[float],
+    max_jobs: Optional[int],
```

Comment:
Hmm, you know I thought we set this to float because -1 was infinite jobs and -1 was a float... but checking it its an int. Nice fix, but can you make sure -1 works for max_jobs?